desktop-tool/src/constants.py

```diff
@@ -181,3 +181,8 @@ class TargetSites(Enum):
 THREADS = 5  # shared between CardImageCollections
 
 POST_LAUNCH_HTML_FILENAME = "post-launch.html"
+
+# Card physical dimensions for border calculation
+CARD_WIDTH_INCHES = 2.5
```

Comment:
```suggestion
CARD_WIDTH_INCHES = 2.48
CARD_HEIGHT_INCHES = 3.46
BORDER_INCHES = 0.12
```
these are the actual values that MPC uses (or at least a bit closer)